bin/cmd.js

```diff
@@ -76,6 +83,12 @@ if (argv.reporter) {
   })
 }
 
+var lintOpts = {}
+if (argv.es6)
```

Comment:
Yeah, I want to change that but it makes pulling in upstream changes a manual process.